enabler/src/de/schildbach/pte/FranceNorthEastProvider.java

```diff
@@ -53,11 +53,21 @@ protected Style getLineStyle(final String network, final Product product, final
         switch (product) {
         case REGIONAL_TRAIN: {
             // Rail (route_type = 2) for TER and Corail Intercité/Lunéa (all SNCF)
-            return new Style(Style.parseColor(color), computeForegroundColor(color));
+            Style s;
+            if(network.equals("SNCF")) // SNCF data have no color
+                s =  new Style(Style.parseColor("#D90283"), computeForegroundColor("#D90283"));
+            else
+                s =  new Style(Style.parseColor(color), computeForegroundColor(color));
+            return s;
         }
         case SUBURBAN_TRAIN: {
             // Rail (route_type = 2) for Transilien (SNCF)
-            return new Style(Style.parseColor(color), computeForegroundColor(color));
+            Style s;
+            if(network.equals("SNCF")) // SNCF data have no color
+                s =  new Style(Style.parseColor("#D90283"), computeForegroundColor("#D90283"));
+            else
+                s =  new Style(Style.parseColor(color), computeForegroundColor(color));
```

Comment:
What happens if some other network that is not SNCF also misses colors? Wouldn't it be better to have a more general situation and prevent crashes whenever a color is missing?

Reply:
Where does this #d90283 color stem from? Generally, if we don't know an "official" color/style for a line, we return null so that the app can decide on how to handle this (e.g. apply default color/style).

Reply:
It is only an arbitrary color inspired by SNCF logo. There is a similar pull request to this problem.
It is just a patch to bypass this NPE and we can imagine the provider manage the default color and not only for the SNCF but I agree it is not perfect, this discussion is linked with #127